Godeps/Godeps.json

```diff
@@ -490,7 +490,7 @@
 		},
 		{
 			"ImportPath": "github.com/container-storage-interface/spec/lib/go/csi/v0",
-			"Comment": "v0.2.0",
+			"Comment": "v0.2.0-rc1",
```

Comment:
Maybe rebase? This looks like it went backwards.

Reply:
Will try, but it may be because it's something kube-openapi depends on

Reply:
Not sure exactly how this is becoming rc1 because https://github.com/jennybuckley/kubernetes/commit/51130376273a328d21c5bfb278829ddc4ee3ab17 updated it straight to v0.2.0 from v0.1.0-19-g31c1670 sometime in march

Reply:
Let me find someone to explain this to us.

Reply:
it looks like the actual revision is the same (35d9f9d77954980e449e52c3f3e43c21bd8171f5), and the tag v0.2.0-rc1 and v0.2.0 both point at that commit (looking at https://github.com/container-storage-interface/spec/releases), so it's not a particularly harmful change, I think.

@jennybuckley how did you save godeps? maybe the GOPATH you're using is out of date?

Reply:
@ixdy 
I saved them with hack/godep-save.sh
Also, I thought GOPATH was just an environment variable pointing to the where the src directory is located.

Anyway, for this PR, since all the generated changes in this file are to the comment and not the revision hash, I just removed them all manually. Thanks for taking a look at this